src/AI/mcp/server/tools/layout_components_tool/layout_components_tool_no_llm.py

```
@@ -0,0 +1,185 @@
+
+## After Using This Tool
+- Use `datamodel_tool` to create data bindings for form components
+- Use `resource_tool` to create text resources for labels
+""",
+    title="Layout Components Tool",
+    annotations=ToolAnnotations(
+        title="Layout Components Tool",
+        readOnlyHint=True
+    )
+)
+def layout_components_tool_no_llm(user_goal: str) -> dict:
```

Ruff (0.15.1)

[warning] Unused function argument: `user_goal`

(ARG001)